src/pages/index.tsx

```diff
@@ -43,7 +46,7 @@ import Tus from '@uppy/tus'
 
 const uppy = new Uppy()
   .use(Dashboard, { target: '.DashboardContainer', inline: true })
-  .use(RemoteSources, { companionUrl: "http://companion.uppy.io" })
+  .use(RemoteSources, { companionUrl: "https://companion.uppy.io" })
```

Comment:
Shouldn’t this use the above companionUrl variable?

Reply:
It's a string, so wanted to be more explicit, but sure, changed.